client/modules/IDE/pages/IDEView.jsx

```diff
@@ -182,6 +193,27 @@ class IDEView extends React.Component {
     }
   }
 
+  updatePaneOrientation() {
+    const minHorizontalWidth = 600;
```

Comment:
ideally, the panel moving would be done with CSS only. in this case, it might not be possible. what you could do, however, is change the orientation from horizontal to vertical (can you use the smallest Bootstrap breakpoint for this?) and then add a class, which would be used to change the CSS.

Reply:
I agree, CSS was my first thought as well. I went down this route after not having much luck with CSS but I can investigate further.

Reply:
for sure! i think you'll find it's easier if you add classes with JS